data_steward/utils/bq.py

```diff
@@ -155,7 +155,7 @@ def get_dataset(project_id, dataset_id):
     return client.get_dataset(dataset_id)
 
 
-def define_dataset(project_id, dataset_id, description, labels):
+def define_dataset(project_id, dataset_id, description, label_or_tag):
```

Comment:
please add `label_or_tag` param to doc_string and please explain the label/tag distinction.